src/betterproto/__init__.py

```diff
@@ -1041,6 +1085,9 @@ def to_dict(
                         ]
                     if value or include_default_values:
                         output[cased_name] = value
+                elif value is None:
+                    if include_default_values:
```

Comment:
This might be better if you merge these if conditions

Reply:
Not quite the same, since that would allow cases where `value is None and include_default_values is False` to fall through to later branches. I think I like the clarity of intent here, but is there something I'm missing?

Reply:
Oh yeah, you're totally right